pkg/storaged/storage-controls.jsx

```diff
@@ -109,16 +109,21 @@ class StorageButton extends React.Component {
         else
             classes += " btn-default";
 
-        return (
-            <StorageControl excuse={this.props.excuse}
-                            content={(excuse) => (
-                                <button id={this.props.id}
-                                            onClick={checked(this.props.onClick)}
-                                            className={classes + (excuse ? " disabled" : "")}>
-                                    {this.props.children}
-                                </button>
-                            )} />
+        let button = (
+            <button id={this.props.id}
+                    onClick={checked(this.props.onClick)}
+                    className={classes + (this.props.excuse ? " disabled" : "")}>
+                {this.props.children}
+            </button>
         );
+        if (!this.props.excuse) {
+            return button;
+        } else {
+            return (
+                <StorageControl excuse={this.props.excuse}
+                                content={() => button} />
+            );
+        }
```

Comment:
Thinking about this again, it would make more sense to fix this centrally in pkg/storaged/storage-controls.jsx 's `render()`. If there is no `excuse`, then this woudl simply not render the `OverlayTrigger` at all. This should fix it for all other instances, too?

Reply:
Makes sense. Reposting.